pages/common/git-merge-base.md

```diff
@@ -0,0 +1,16 @@
+# git merge-base
+
+> Find a common ancestor of two commits.
+> More information: <https://git-scm.com/docs/git-merge-base>.
+
+- Output a best common ancestor of two commits:
+
+`git merge-base {{commit_1}} {{commit_2}}`
+
+- Output all best common ancestors of two commits:
```

Comment:
`best` can only be 1.
```suggestion
- Print all merge bases for two commits:
```

Reply:
In the way git weighs merge bases, there actually can be two "best" ones, which means two of the same quality. But I agree that the wording is confusing, so maybe "all suitable" should be used?

Reply:
Ok